diff --git a/custom_components/verisure/__init__.py b/custom_components/verisure/__init__.py
--- a/custom_components/verisure/__init__.py
+++ b/custom_components/verisure/__init__.py
@@ -63,13 +63,16 @@
     if not unload_ok:
         return False
 
+    return True
+
+
+async def async_remove_entry(hass: HomeAssistant, entry: VerisureConfigEntry) -> None:
+    """Erase session cookie when the config entry is deleted."""
     cookie_file = hass.config.path(STORAGE_DIR, f"verisure_{entry.data[CONF_EMAIL]}")
     with suppress(FileNotFoundError):
         await hass.async_add_executor_job(os.unlink, cookie_file)
 
-    return True
 
-
 def migrate_cookie_files(hass: HomeAssistant, entry: ConfigEntry) -> None:
     """Migrate old cookie file to new location."""
     cookie_file = Path(hass.config.path(STORAGE_DIR, f"verisure_{entry.unique_id}"))
